src/helper/client.cpp

```diff
@@ -433,6 +433,13 @@ bool client::handle_command(network::request_shutdown::request &command)
 
     auto response = std::make_shared<network::request_shutdown::response>();
     try {
+        if (context_->schema_extraction_sampled()) {
```

Comment:
This doesn't really work, the sampler scope will go out of scope before you call publish.

Reply:
Done